rules/cloud/m365/audit/microsoft365_teams_guest_rmm_deployment.yml

```
@@ -0,0 +1,56 @@
+title: Detection of Potential Malicious Guest Accounts and RMM Tool Deployment via Teams
```

We don't use the keyword "Detection" in titles as it is highlighted by the SigmaHQ convention

```suggestion
title: Potential Malicious Guest Accounts and RMM Tool Deployment via Teams
```